libs/wallet/src/connect-dialog/connect-dialog.tsx

```diff
@@ -306,77 +300,162 @@ const SelectedForm = ({
     );
   }
 
-  if (connector instanceof ViewConnector) {
-    return (
-      <ViewConnectorForm
-        connector={connector}
-        onConnect={onConnect}
-        reset={reset}
-      />
-    );
-  }
-
   throw new Error('No connector selected');
 };
 
+const GetWallet = () => {
+  const isItChrome = window.navigator.userAgent.includes('Chrome');
+  const isItMozilla =
+    window.navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
+  const onClick = () => {
+    if (isItMozilla) {
+      window.open(MOZILLA_EXTENSION_URL, '_blank');
+      return;
+    }
+    if (isItChrome) {
+      window.open(CHROME_EXTENSION_URL, '_blank');
+    }
+  };
+
+  const buttonContent = (
+    <>
+      <div className="flex items-center justify-center gap-1 text-base">
+        {t('Get the Vega Wallet')}
+        <Lozenge className="text-[10px] !font-alpha bg-vega-blue-500 dark:bg-vega-blue-500 py-0 px-1">
```

Comment:
You're basically overriding every single style defined for the Lozenge component. I would suggest just creating a new Pill component for trading only that follows the same intent pattern used for buttons/notifications

Reply:
This overriding turned out as not necessary anymore, because I made other solution with html inside the button. Anyway "fill" property would be needed, so I'm gonna add it

Reply:
component Pill has been added